package.json

```diff
@@ -21,6 +21,7 @@
   "browser": {
     "./lib/swarm-mini/utils/data.js": "./lib/swarm-mini/utils/data.browser.js",
     "./src/swarm-mini/utils/data.ts": "./src/swarm-mini/utils/data.browser.ts",
+    "./test/providers/ipfs/create-repo-nodejs.js": "./test/providers/ipfs/create-repo-browser.js",
```

Comment:
Why we should include test here?

Reply:
Because we test in Browser and in there the `create-repo-browser.js` should be used instead.